models/buildCluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Command = require('../config/command');
+const scmContext = Joi.reach(require('./pipeline').base, 'scmContext');
+const scmOrgization = Joi
+    .string().max(100)
+    .description('SCM organization name')
+    .example('screwdriver-cd');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .description('Identifier of this Job')
+        .example(123345),
+
+    name: Joi
+        .string().regex(/^[\w-]+$/)
+        .max(50)
+        .description('Name of the build cluster')
+        .example('iOS'),
+
+    description: Joi
+        .string().max(100)
+        .description('Description of the build cluster')
+        .example('Build cluster for iOS team'),
+
+    scmContext,
+
+    scmOrganizations: Joi.array().items(scmOrgization),
+
+    isActive: Joi
```

Comment:
is this necessary?

Reply:
yeah, e.g. when we have 2 clusters and want to temporarily turn off one for maintainence